### DISCUSSION TITLE
Page title — имеет двусмысленность

### DISCUSSION THREAD

**Discussion by Vivioza5** [open-tutorials/cypress#40] [Ideas]
Текстовая версия описания тест кейса open_article_detail_page Tutorial:finish mama project [пункт 7](https://github.com/open-tutorials/cypress/blob/8e0aafcf39e221f0e98954d7208dbf29f37bfbd0/test_cases/articles/global_feed/open_article_detail_page.md?plain=1#L19) имеет проверку:

7. **Page title** should be `{random_article_title}`

 Cчитаю что при поиске этого элемента можно столкнуться с тем, что  будешь искать именно название страницы, который прописан в `<head>`, может стоит переназвать например "Article name on page"

**Comment by Vladimir-Tarasov** [comment]
Думаю с этим проблем быть не должно так как мы проверяем содержимое страницы находящееся в `<body>`.

**Reply by breslavsky** [reply]
Как насчет того, что бы написать?

```md
Page heading should be {random_article_title}
```

**Reply by breslavsky** [reply]
Исправил везде.
